k2/csrc/fsa_equivalent.h

```diff
@@ -60,6 +60,49 @@ bool IsRandEquivalent(const Fsa &a, const float *a_weights, const Fsa &b,
                       float delta = 1e-6, bool top_sorted = true,
                       std::size_t npath = 100);
 
+/*
```

Comment:
Put this function in file `fsa_equivalent.h` as:
- It may be easy for user to check these similar `RandEquivalent` functions together.
- We may need to refactor those functions to extract some common sub-functions (such as ::GetArcWeights that should not be in k2 namespace), it's easy to call those sub-functions if we put these `RandEquivalent` functions in one file.

Reply:
cool.